<launch>
    <arg name="map_file" default="$(find course_agv_gazebo)/config/map.yaml"/>
    <!-- Run the map server -->
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />

	<node pkg="course_agv_slam" type="localization.py" name="icp" output="screen">
		
		<!--from a rosbag-->
		<param name="robot_x" value="0"/>
		<param name="robot_y" value="0"/>
		<param name="robot_theta" value="0"/>

		<param name="max_iter" value="10"/>
		<param name="tolerance" value="0"/>
		<param name="dis_th" value="3"/>
	</node>
	<node pkg="course_agv_slam" type="wheel.py" name="wheel_odometry" output="screen" launch-prefix="xterm -e" />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find course_agv_slam)/launch/course_agv.rviz" />
</launch>
